ntoskrnl/rtl/libsupp.c

```diff
@@ -691,6 +691,67 @@ RtlpGetAtomEntry(PRTL_ATOM_TABLE AtomTable, ULONG Index)
    return Entry;
 }
 
+/* Ldr SEH-Protected access to IMAGE_NT_HEADERS */
+
+/* Rtl SEH-Free version of this */
+NTSTATUS
+NTAPI
+RtlpImageNtHeaderEx(
+    _In_ ULONG Flags,
+    _In_ PVOID Base,
+    _In_ ULONG64 Size,
+    _Out_ PIMAGE_NT_HEADERS *OutHeaders);
+
+#ifndef RTL_IMAGE_NT_HEADER_EX_FLAG_NO_RANGE_CHECK
+#define RTL_IMAGE_NT_HEADER_EX_FLAG_NO_RANGE_CHECK          0x00000001
+#endif
```

Comment:
Do we not always have NDK included here?

Reply:
The symbol was not available, would you prefer I try to include the correct header?

Reply:
It is defined under `NTOS_MODE_USER`

Reply:
this also causes a problem in freeldr

Reply:
Hrm. Since this function clearly exists in kernel mode, it should be fine to move the define out of `NTOS_MODE_USER` in NDK.
But if that causes trouble elsewhere, the redefine here seems reasonable.